docs/components/lists.md

```diff
@@ -255,40 +262,44 @@ ListView(
 
 The following are tables of the list item contents:
 
+1. Supporting visuals
+1. Primary text
+1. Secondary text
+1. Tertiary text
+1. Metadata
+1. Controls
 
-##### Supporting visuals
+#### Supporting visuals for Three Line List
```

Comment:
FYI, Google titles use sentence case:

"This is an example of a Google title"
"This is Not an Example of a Google Title"

Reply:
Okay